src/RNGestureHandlerModule.ts

```diff
@@ -3,3 +3,27 @@
 
 import Module from './specs/NativeRNGestureHandlerModule';
 export default Module;
+
+declare const global: {
+  _WORKLET_RUNTIME: ArrayBuffer;
+};
+
+function tryInstallUIBindings() {
+  if (global._WORKLET_RUNTIME) {
+    console.log('Installing Gesture Handler UI bindings');
```

Comment:
Do we want to keep those console.logs in `.ts` file? Maybe let's convert them to comments.

Reply:
Nope, but they are useful for development. One of the todos from the PR description is `get rid of the console logs`